content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,279 @@
+
+### Make Kubernetes aware of the LoadBalancer behaviour ([SIG Network](https://github.com/kubernetes/community/tree/master/sig-network))
+
+The `LoadBalancerIPMode` feature gate is now beta and is now enabled by default. This feature allows you to set the `.status.loadBalancer.ingress.ipMode` for a Service with `type` set to `LoadBalancer`. The `.status.loadBalancer.ingress.ipMode` specifies how the load-balancer IP behaves. It may be specified only when the `.status.loadBalancer.ingress.ip` field is also specified. See more details about [specifying IPMode of load balancer status](/docs/concepts/services-networking/service/#load-balancer-ip-mode).
+
+## New alpha features
+
+### Speed up recursive SELinux label change ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+From the v1.27 release, Kubernetes already included an optimization that sets SELinux labels
+on the contents of volumes, using only constant time.
+Kubernetes achieves that speed up using a mount option. The slower legacy behavior requires
+the container runtime to recursively walk through the whole volumes and apply SELinux labelling
+individually to each file and directory; this is especially noticable for volumes with large amount of
+files and directories.
+
+Kubernetes 1.27 graduated this feature as beta, but limited it to ReadWriteOncePod volumes.
+The corresponding feature gate is `SELinuxMountReadWriteOncePod`. It's still enabled by default
+and remains beta in 1.30.
+
+Kubernetes 1.30 extends support for SELinux mount option to **all** volumes as alpha,
+with a separate feature gate: `SELinuxMount`. This feature gate introduces a behavioral change
+when multiple Pods with different SELinux labels share the same volume.
+See [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1710-selinux-relabeling#behavioral-changes) for details.
+We strongly encourage users that run Kubernetes with SELinux enabled to test this feature and
+provide any feedback on the [KEP issue](https://kep.k8s.io/1710).
+
+| Feature gate | Stage in v1.30 | Behavior change |
+|---|---|---| 
+| SELinuxMountReadWriteOncePod |  Beta | No |
+| SELinuxMount | Alpha | Yes | 
+
+Both feature gates `SELinuxMountReadWriteOncePod` and `SELinuxMount` must be enabled to test this feature on all volumes.
+
+This feature has no effect on Windows nodes or on Linux nodes without SELinux support.
+
+### Recursive Read-only (RRO) mounts ([SIG Node](https://github.com/kubernetes/community/tree/master/sig-node))
+
+Blurb for 3857 goes here--subject to Docs Freeze
+
+### Job success/completion policy ([SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps))
+
+Blurb for 3998 goes here--subject to Docs Freeze
```

Comment:
```suggestion
From Kubernetes v1.30, indexed Jobs support `.spec.successPolicy` to define when a Job can be declared succeeded based on succeeded Pods. This allows you to define two types of criteria:

- `succeededIndexes` indicates that the Job can be declared succeeded when these indexes succeeded, even if other indexes failed.
- `succeededCount` indicates that the Job can be declared succeeded when the number of succeeded Indexes reaches this criterion.

After the Job meets the success policy, the Job controller terminates the lingering Pods.
```

Reply:
@kcmartin @sftim please note: Yuki (@tenzen-y) has had a problem with their GitHub account (currently being reviewed by the GitHub team), so the KEP for this feature isn't currently available. Yuki has kindly sent this blurb for the KEP via Slack, and I've done my best to grammar edit. Your review is also welcome, thanks!